DEV-GUIDE.md

```diff
@@ -195,6 +199,58 @@ undesirable as Golang may not erase out of date artifacts from the
 cache. In such a situation, erase the contents of `$GOPATH/pkg/` manually
 before rebuilding the code.
 
+### Building individual Istio components
+
+You can build individual Istio components by using their corresponding build
+targets. The following targets are currently defined:
+
+* pilot: build the pilot-discovery executable
+* proxy: build the pilot-agent executable
+* istioctl: build the istioctl executable
+* sidecar_initializer: build the istio-initializer executable
+* mixs: build the mixer server execuable
+* mixc: build the mixer client executable
+* servicegraph: build the servicegraph executable
+* node-agent: build the node_agent executable
+* istio-ca: build the istio_ca executable
+
+For example, you can build pilot-discovery and pilot-agent with the following
+command:
+
+```shell
+make pilot proxy
+```
+
+### Building the code with debugger information
```

Comment:
And regardless of previous ( if we do individual components) - this part is completely not needed.

It may be ok to say DEBUG=1 - but having debug.pilot and all the other combinations is far too much.

Reply:
Adding debug.pilot, etc is a simple addition following the existing support of building individual component as indicated in last comment.
I think that it should be useful for non-containerized components such as istioctl and other client components.

Reply:
Add an env variable - DEBUG=1 or whatever - and then all targets will be built in debug mode.

There is no need to add a all possible permutations of options as additional targets just because there are some already.